src/oidcendpoint/oidc/add_on/pkce.py

```diff
@@ -124,21 +121,7 @@ def post_token_parse(request, client_id, endpoint_context, **kwargs):
 
 
 def add_pkce_support(endpoint, **kwargs):
-    authn_endpoint = endpoint.get("authorization")
-    if authn_endpoint is None:
-        LOGGER.warning(
-            "No authorization endpoint found, skipping PKCE configuration"
-        )
-        return
-
-    token_endpoint = endpoint.get("token")
-    if token_endpoint is None:
-        LOGGER.warning(
-            "No token endpoint found, skipping PKCE configuration"
-        )
-        return
-
-    authn_endpoint.post_parse_request.append(post_authn_parse)
```

Comment:
Why was this removed?

Reply:
probably a better path for this module would be `src/oidcendpoint/oauth2/add_on/pkce.py` as it belongs to OAuth2 and can be used even in oidc

Reply:
Right!